Prueba1.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumTests
+{
+    [TestFixture]
+    public class Pueba1
```

Comment:
Please use just one language, `English` or `Spanish`. Also, you're missing the `r` in `Pueba1`.